### PR TITLE
fix(valibot-validator): update Hono version and fix test code

### PR DESCRIPTION
**Pull request by nakasyou**
I changed the test code to match the new Hono version.

### CODE REVIEW

File: .changeset/plenty-pandas-press.md
Comment (yusukebe) [inline .changeset/plenty-pandas-press.md:1]: We don't have to add the changeset if the change is only for testing. The changeset should only affect things for users. If you add a changeset, the version will be incremented.

**Comment by yusukebe** [comment]
@nakasyou Thank you for the PR! I've left a comment.

File: .changeset/plenty-pandas-press.md
Comment (nakasyou) [reply .changeset/plenty-pandas-press.md:1]: I did it.

**Comment by yusukebe** [comment]
@nakasyou 

Thanks. Merging.

**Pull request merged**